nservicebus/operations/auditing.md

```diff
@@ -77,6 +77,8 @@ snippet:AuditWithCode
 
 ### Using app.config
 
+Warning: Configuration of auditing via `app.config` is not recommended in Version 6, use the [code configuration API](#configuring-auditing-using-code) instead. The configuration section will be removed in Version 7.
```

Comment:
Done @SimonCropp. We lose the links to the code section, but I think that's fine. These warnings are all placed close enough to the code configuration sample to be on the same screen.